databricks/koalas/indexes.py

```diff
@@ -1312,6 +1312,112 @@ def levshape(self):
         result = internal._sdf.agg(*(F.countDistinct(c) for c in internal.index_scols)).collect()[0]
         return tuple(result)
 
+    @property
+    def is_monotonic(self):
```

Comment:
We don't need to override this too. Just overriding `_is_monotonic` should be fine.

Reply:
Let's have the doctests in the super class.